lib/setup.js

```diff
@@ -198,6 +199,9 @@ export default function (app) {
   app.use(unsupportedBrowserCheck)
   app.use(splitTestMiddleware)
 
+  // Sets up mobile marketing signup modal
+  app.use(marketingSignupModal)
```

Comment:
This what was missing in order to setup the marketing signup model for mobile. Is there a better place to put this?